microsoft-edge/progressive-web-apps-chromium/how-to/handle-protocols.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Handle protocols in Progressive Web Apps
+description: Learn how to register your PWA as a protocol handler to more deeply integrate it in the operating system with other applications.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/29/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+keywords: progressive web apps, PWA, Edge, JavaScript, protocol
+---
+# Handle protocols in Progressive Web Apps
+
+Progressive Web Apps can handle protocols to create a more engaging experience.
+
+A URL (or Uniform Resource Locator) is a string that allows clients to identify a resource on the Web, and it contains an initial part, the protocol, that lets clients know how to retrieve the resource.
+
+Examples of protocols include:
+
+| Protocol | Example |
+|:--- |:--- |
+| `http` | http://contoso.com/ |
```

Comment:
I don't think we want to give the impression that a PWA can register for any scheme. There is an allowed list, which can change at any time. (In reality, its only ever expanded). But they can always register for web+<something>

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/custom_handlers/protocol_handler_utils.cc;l=40